### ISSUE TITLE
error message getting token from access server

### ISSUE THREAD

**Issue by demon740** [vpnhood/VpnHood#381]
I had a fresh install with command that I have got from https://console.vpnhood.com.
after installation I can see server is IDLE in the VpnHood Access Server.

then I create an access token on that server and I want to get its "Access Key" but this error happens:
Error
Could not find any public access point for the ServerFarm. Please configure a server for this AccessToken.
Details:
{
  "Data": {},
  "TypeName": "InvalidOperationException",
  "TypeFullName": "System.InvalidOperationException",
  "Message": "Could not find any public access point for the ServerFarm. Please configure a server for this AccessToken."
}

I have an old server that suddenly I cannot connect to it or get access key for previous created tokens, and I have installed a fresh server that has this problem too.

thanks

**Comment by trudyhood** [comment]
It means your farm does not have any server or your server does not have any "PublicInToknen" in its access points, 
![image](https://github.com/vpnhood/VpnHood/assets/72891124/a9f51df0-3748-4371-ba31-f53149cb0f9a)

**Comment by demon740** [comment]
Thanks for your quick answer. As I said, I have an old server and this happens suddenly for it and after that, I created a new one from base, a new Linux and new installation. I got installation command from [console.vpnhood.com], but it happened for the new installation again.

My question is: what is this PublicInToken and how this happened for me when I got full installation procedure from [console.vpnhood.com].

Thanks.
